default/AI/ColonisationAI.py

```diff
@@ -980,13 +993,16 @@ def evaluate_planet(planet_id, mission_type, spec_name, empire, detail=None):
         if have_existing_presence:
             detail.append("preexisting system colony")
             retval *= 1.5
+        supplyval = 0
         if sys_supply < 0:
             if sys_supply + planet_supply >= 0:
-                retval += 30 * (planet_supply - max(-3, sys_supply))
+                supplyval += 30 * (planet_supply - max(-3, sys_supply))
```

Comment:
This meant "How about change supplyval to supply_val?"  :) 
This name have different style then neighbor variables.  It looks like single word that has typo.

PS. There are many words in your answer, it to hard reading for this night. This name don`t worth additional discussion.